### PR TITLE
382 notebook review

### LINKED ISSUE #382: Notebook Review #4
1. 2.3 https://github.com/Qiskit/qiskit-metal/tree/main/tutorials/2%20Front%20End%20User/2.3%20My%20first%20QDesign

2. 2 QComponent Designer: https://github.com/Qiskit/qiskit-metal/tree/main/tutorials/3%20QComponent%20Designer

3. ^

### PR DESCRIPTION
**Pull request by smukesh25**
### What are the issues this pull addresses (issue numbers / links)?
Closes #382 

### Did you add tests to cover your changes (yes/no)?

Yes
### Did you update the documentation accordingly (yes/no)?

Yes
### Did you read the CONTRIBUTING document (yes/no)?

Yes

### Summary

Reviewed Notebooks corrected spellings, removed redundant analyses.

### CODE REVIEW

**Comment by grace-harper-ibm** [comment]
Added comments to help clarify using Ansys
Priti and I added these comments to the 8 notebook. Let me know if you think they shouldn't be there and I'll remove them.

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
LGTM

File: tutorials/2 Front End User/2.3 My first QDesign/Algorithmic_Chip_Design.ipynb [hunk 1]
```diff
@@ -199,7 +199,44 @@
    "metadata": {},
    "source": [
     "We can quickly check if we have designed the qubits well with an LOM analysis. \n",
-    "First we render the qubit to Ansys Q3D, presuming we have already opened Ansys."
+    "First we render the qubit to Ansys Q3D. Comment the \"LOM_q3d.open_ansys()\" command if you already have an Ansys session open."
```
Comment (priti-ashvin-shah-ibm) [inline tutorials/2 Front End User/2.3 My first QDesign/Algorithmic_Chip_Design.ipynb:202]: At the top of the notebook, please add a cell in "Markdown".

Suggestion:
# Prerequisite
You need to have a working local installation of Ansys

**Review by JeremyDrysdale-ibm — DISMISSED** [review]
LGTM

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Review by grace-harper-ibm — APPROVED** [review]
LGTM

**Review by JeremyDrysdale-ibm — APPROVED** [review]
LGTM

**Pull request merged**